mkdocs/docs/HPC/connecting.md

```diff
@@ -474,7 +474,7 @@ again on your local machine:
 </code></pre>
 
 You can now log out, open a new terminal/shell on your local machine and
-reconnect to the , and you should not get these warnings anymore.
+reconnect to the login node, and you should not get these warnings anymore.
```

Comment:
The indent is off here as well, this paragraph (and the ones above) all belong in the "`tip: Setting your Language right`", so fix that too while you're at it?